pkg/provider/cloud/kubevirt/provider.go

```diff
@@ -130,7 +130,7 @@ func (k *kubevirt) reconcileCluster(ctx context.Context, cluster *kubermaticv1.C
 		return cluster, err
 	}
 
-	err = reconcileInfraTokenAccess(ctx, cluster.Status.NamespaceName, client)
+	err = ReconcileInfraTokenAccess(ctx, cluster.Status.NamespaceName, client)
```

Comment:
just a nit, please inside the function change the SA name that is taken from
`resource.KubeVirtCSIServiceAccountName` instead of `csiResourceName`

Reply:
basically we should replace constant `csiResourceName` with `resource.KubeVirtCSIServiceAccountName` in the whole CSI file.

Reply:
done